app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
```

Comment:
While generally I would agree, the multi-line is more clear here IMO. My general rule of thumb is to not compound multiple expressions on the same line. 

```
render text: 'Not Found', status: :not_found if @game.blank?
```

In the above, it is not clear from a glance where the method ends and the condition begins. 

In other languages this would probably be fine, but due to the Ruby style point of not using `(` and `)` for most methods, we loose the clarity. 

The below would be more clear, but it would break the no parenthesis styling
```
render (text: 'Not Found', status: :not_found) if @game.blank?
```